### DISCUSSION TITLE
Dolphin 24

### DISCUSSION THREAD

**Discussion by Senshi111** [prasanthrangan/hyprdots#576] [Q&A]
Are you guys having problem with dolphin 24 version and sytle?  qt5ct will not accept anything from styles, qt6ct works half way, but won't accept colors or change style when changing theme

**Comment by kRHYME7** [comment]
Hi, How can I download Version 24? On Arch, I could only see version 23 ATM.

```
❯ paru -Qs dolphin
local/dolphin 23.08.3-1 (kde-applications kde-system)
    KDE File Manager ```

**Comment by Senshi111** [comment]
I'm on fedora rawhide at the moment. I made this theme to work with debian sid and fedora.

**Comment by Senshi111** [comment]
![231201_22h48m28s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/40372242/d6f95c9e-4f84-4bcf-bbe3-b3a3d50201c8)
![231201_22h49m21s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/40372242/ab504122-2787-416d-97fb-8f84979facfa)
I think we will have some problems with dolphin 24 version
